_example/users/delete_user.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"log"
+	"os"
+
+	"github.com/zoom-lib-golang/zoom-lib-golang"
+)
+
+func main() {
+	var (
+		apiKey    = os.Getenv("ZOOM_API_KEY")
+		apiSecret = os.Getenv("ZOOM_API_SECRET")
+		userID    = os.Getenv("USER_ID")
+	)
+
+	zoom.APIKey = apiKey
+	zoom.APISecret = apiSecret
+
+	// Delete User by UserID or Email
+	opt := zoom.DeleteUserOptions{
+		EmailOrID: userID,
+	}
+
+	response := zoom.DeleteUser(opt)
+
+	log.Printf("Resopnse: %+v\n", response)
```

Comment:
Calling the return value `response` is a bit misleading IMO as it is not a response object (i.e. a user), the return value is only error. 

Would like to see it more like this:

```golang
if err := zoom.DeleteUser(opt); err != nil {
    log.Printf("Error: %+v\n", err)
}